app/assets/stylesheets/custom.scss

```
@@ -2,8 +2,19 @@
 //
 // You can also create SCSS files in the app/assets/stylesheets/custom/
 // folder and they will be loaded automatically.
-.legislation-hero {
-  .prompt {
+
+.section-prompt {
+  @include full-width-background;
+  @include grid-column-gutter;
+
+  filter: brightness(0.95);
+  margin-top: -$line-height;
+  margin-bottom: $line-height;
+  padding-bottom: $line-height / 2;
```

If we intended to have similar padding on top and bottom:

```suggestion
  padding-bottom: $line-height;
```

And then we add `margin-bottom: 0` to the `p` child.